### PR TITLE
[DOC] Fix controllerFor typo

### PR DESCRIPTION
**Pull request by yogipatel**
`Ember.route#controllerFor` was documented as returning a model instead of a controller. This PR corrects that.

### CODE REVIEW

File: packages/ember-routing/lib/system/route.js [hunk 1]
```diff
@@ -1762,7 +1762,7 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
   },
 
   /**
-    Returns the resolved model of the current route, or a parent (or any ancestor)
+    Returns the resolved controller of the current route, or a parent (or any ancestor)
```
Comment (rwjblue) [inline packages/ember-routing/lib/system/route.js]: I realize this isn't really related to your change, but we should probably remove `resolved` from here.  We don't do any promise resolution, and it looks like this was just copy/pasta'ed from `modelFor` (which does resolve promisey things).

**tldr;** Would you mind removing `resolved` here?

Comment (yogipatel) [inline packages/ember-routing/lib/system/route.js]: 👍 I was wondering the same thing.

**Comment by mmun** [comment]
Thank you.
